secrets.yml

```diff
@@ -9,3 +9,5 @@ production:
 
   AWS_ACCESS_KEY_ID: !var prod/aws/iam/users/secretless.io-publisher-prod/access_key_id
   AWS_SECRET_ACCESS_KEY: !var prod/aws/iam/users/secretless.io-publisher-prod/secret_access_key
+
+REDHAT_API_KEY: !var redhat/projects/secretless-broker/api-key
```

Comment:
can this be moved to the top in a `common` section, like we do [here](https://github.com/conjurdemos/kubernetes-conjur-demo/blob/master/ci/secrets.yml)? otherwise I think it's possible this might not work

Reply:
Done